package.json

```diff
@@ -1,24 +1,38 @@
 {
   "name": "@stylelint/prettier-config",
   "version": "2.0.0",
-  "description": "stylelint org's shareable config for prettier",
+  "description": "A stylelint shareable config for Prettier.",
   "keywords": [
+    "@stylelint",
     "prettier",
     "prettierconfig"
   ],
+  "homepage": "https://github.com/stylelint/prettier/tree/master/packages/prettier-config#readme",
```

Comment:
@hudochenkov mentioned in another thread that neither `homepage` or `bugs` are required when using `repository` shorthand, e.g. `"repository": "stylelint/prettier-config`. I've been stripping them out of the other repositories.

Reply:
Thanks, fixed in 2.0.0 release https://github.com/stylelint/npm-package-json-lint-config/releases/tag/2.0.0